src/libs/API.js

```diff
@@ -741,13 +741,13 @@ function User_GetBetas() {
 
 /**
  * @param {Object} parameters
- * @param {String} parameters.email
+ * @param {Array} parameters.emailList
  * @param {Boolean} [parameters.requireCertainty]
  * @returns {Promise}
  */
 function User_IsFromPublicDomain(parameters) {
     const commandName = 'User_IsFromPublicDomain';
-    requireParameters(['email'], parameters, commandName);
+    requireParameters(['emailList'], parameters, commandName);
     return Network.post(commandName, {
         ...{requireCertainty: true},
         ...parameters,
```

Comment:
Am I mistaken in thinking that `User_IsFromPublicDomain` expects a comma-separated list of emails instead of an array? Should this be something like:

```js
return Network.post(commandName, {
    ...{
        requireCertainty: true,
        emailList: parameters.emailList.join(','),
    },
    ...(_.omit(parameters, 'emailList')),
});
```

Or perhaps more simply:

```js
return Network.post(commandName, {
    ...parameters,
    ...{
        requireCertainty: true,
        emailList: parameters.emailList.join(','),
    },
});
```

The second is better as long as we want `requireCertainty` to always be `true`, which I think we do.

Reply:
You are right! In fact, I'm a bit confused by that too because even though we're passing an array, the request is actually sent as a comma separated list! I can't seem to find where it's being converted though 🤔 

Anyway, I updated the call to `User_IsFromPublicDomain` in `User.js` to make it clearer.

Reply:
NAB but I think it would be a bit cleaner to do the `.join` in here and have it take an array.

Reply:
Pretty sure this happens when you use `FormData`

![2021-09-15_14-08-01](https://user-images.githubusercontent.com/32969087/133528827-732a4a39-0c49-4365-81a7-f826d557da41.png)

But fun fact last time I checked it did not work this way on iOS 😃

Reply:
TIL